tests/test_game.py

```diff
@@ -1,87 +1,49 @@
 import pytest
 from ttt.game import Game
-
-
-class MockBoard:
-    def __init__(self):
-        self.occupy_space_call_count = 0
-        self.get_spaces_call_count = 0
-
-    def is_full(self):
-        return False
-
-    def place_marker(self, space, marker):
-        self.occupy_space_call_count += 1
-
-    def get_spaces(self):
-        self.get_spaces_call_count += 1
-
-    def is_valid_space(self, space):
-        lowest_indexed_space = 0
-        highest_indexed_space = 8
-        occupied_space = 3
-        if space < lowest_indexed_space or space > highest_indexed_space or space == occupied_space:
-            return False
-        else:
-            return True
-
-    def is_winning_line(self, line):
-        return False
-
-
-class MockFullBoard:
-    def is_full(self):
-        return True
-
-    def is_winning_line(self, line):
-        return False
-
-
-class MockBoardWithWinConditionMet:
-    def is_full(self):
-        return False
-
-    def is_winning_line(self, line):
-        return True
+from ttt.console import Console
+from tests.mocks import MockConsoleIO
+from tests.mocks import MockPlayer
+from tests.mocks import MockBoard
 
 
 @pytest.fixture
-def game():
-    return Game(game_board=MockBoard())
+def players():
+    return [MockPlayer('player 1', 'O'), MockPlayer('player 2', 'X')]
 
 
 @pytest.fixture
-def custom_names_game():
-    return Game(game_board=MockBoard(), player_one='!', player_two='@')
-
-
-def test_player_one_can_have_custom_symbol(custom_names_game):
-    assert custom_names_game.get_player_one() == '!'
-
-
-def test_player_two_can_have_custom_symbol(custom_names_game):
-    assert custom_names_game.get_player_two() == '@'
+def game(players):
+    return Game(game_board=MockBoard(), player_one=players[0], player_two=players[1])
 
 
 def test_get_board_state_sends_a_message_to_board():
     board = MockBoard()
-    game = Game(board)
+    game = Game(game_board=board)
     game.get_board_state()
 
     assert board.get_spaces_call_count == 1
 
 
-def test_current_player_switches_to_player_two_after_player_one_turn(game):
-    current_player_after_switch = game.get_player_two()
-    game.play_turn(0)
+def test_available_spaces_sends_a_message_to_board():
```

Comment:
I don't understand why we need this test. It shows me that game delegates to board. As a client of game, do I need to know this?